transport/http/codec.go

```diff
@@ -63,6 +65,10 @@ func DefaultRequestDecoder(r *http.Request, v interface{}) error {
 		return errors.BadRequest("CODEC", fmt.Sprintf("unregister Content-Type: %s", r.Header.Get("Content-Type")))
 	}
 	data, err := io.ReadAll(r.Body)
+
+	// reset body.
+	r.Body = ioutil.NopCloser(bytes.NewBuffer(data))
```

Comment:
Could you add a corresponding unit test for this modification

Reply:
👌🏻